modules/ROOT/pages/general-configuration-guide/basic-setup.adoc

```diff
@@ -86,15 +90,16 @@ tinymce.init({
 });
 ----
 
-For a full list of plugins and their options, see: [Add plugins to TinyMCE](\{\{ site.baseurl }}/plugins/).
+For a full list of plugins and their options, see: xref:plugins.adoc[Add plugins to TinyMCE].
```

Comment:
This xref doesn't resolve, as that file doesn't exist. I believe it should be this:

```suggestion
For a full list of plugins and their options, see: xref:plugins/index.adoc[Add plugins to TinyMCE].
```

- [ ] Fix broken xref

Reply:
Hi @lnewson if this is resolved can you please mark the task as done ? Thanks

Reply:
Done, sorry I must have missed this.